include/swift/Basic/LangOptions.h

```diff
@@ -387,6 +387,9 @@ namespace swift {
     ASTVerifierOverrideKind ASTVerifierOverride =
         ASTVerifierOverrideKind::NoOverride;
 
+    /// Allow @hasAsyncAlternative attribute
+    bool EnableExperimentalHasAsyncAlternative = false;
```

Comment:
I don't think we need a flag for this. `ConcurrencyOnly` ensures that it can only be used when `-enable-experimental-concurrency` is provided.

Reply:
Thanks, wasn't sure whether it being behind concurrency only was okay :). Good to know!